lib/swipe_image_gallery.dart

```diff
@@ -325,8 +325,11 @@ class SwipeImageGallery {
       },
     );
 
+    /// result of navigator
+    T? result;
```

Comment:
Rather than adding a comment for this, could you rename this to be something like `navigatorResult`?
But solid work! I can merge and create a release with this included soon.

Reply:
perfect, thank you